app-tracking-protection/vpn-api/src/main/java/com/duckduckgo/mobile/android/vpn/feature/AppTpSetting.kt

```diff
@@ -22,6 +22,7 @@ enum class AppTpSetting(override val value: String, override val defaultValue: B
     ProtectGames("protectGames"),
     ExceptionLists("exceptionLists", defaultValue = true),
     RestartOnConnectivityLoss("restartOnConnectivityLoss", defaultValue = true),
+    VerboseLogging("verboseLogging"),
```

Comment:
We're trying to deprecate these `AppTpSetting`s (deprecated way to do feature flagging).
 Can we use a feature flag created with `@ContributesRemoteFeature` instead?
You could add it in `:vpn-impl` under `com.duckduckgo.mobile.android.vpn.debug` package. No need to expose it in an `:api` module as `:vpn-internal` has visibility to all `:vpn-impl`

```kotlin
// This feature will never be in remote config
@ContributesRemoteFeature(
    scope = AppScope::class,
    featureName = "appTpLocalFeature",
)
interface AppTpLocalFeature {
    @Toggle.DefaultValue(false)
    fun self(): Toggle

    @Toggle.DefaultValue(false)
    fun verboseLogging(): Toggle
}
```

Reply:
Cool, that's much cleaner, thanks!